configs/satellite_config.yaml

```diff
@@ -247,3 +247,51 @@ pipes:
       client_name: grpc-client
       forwarders:
         - plugin_name: nativemeter-grpc-forwarder
+  - common_config:
```

Comment:
Should the fetcher pipeline be in the document only?
I think in default, we don't scratch from anywhere.

Reply:
OK, will move to docs